pages/blog/posts/json-schema-ambassadors.md

```diff
@@ -0,0 +1,96 @@
+
+Given the prominence of the ambassadors, we will, collectively, check any and all published work they produce, making sure it's accurate. If you work with code, you've probably seen your fair share of poorly documented projects, and you know the impact that can have. High-quality, up-to-date content makes it easier for both newcomers and veterans to learn and stay current.
+
+Further, we don't require or even expect that all the content ambassadors produce will go onto the project's website. No. There's actually improved reach by ambassadors using their existing channels to reach their community, hopefully expanding the JSON Schema community in addition to educating other spaces.
+
+## A portrait of an Ambassador
+
+We've long planned the Ambassadors Program and have identified initial candidates. (You can nominate yourself or others!) Here are the qualities we seek in ambassadors.
+
+Firstly, an ambassador has to be with the community. This means in the slack server, on Github, and occasionally on calls or other places representing or being a recognised expert or source of knowledge for JSON Schema.
+
+Critically, we expect ambassadors to be aligned with the JSON Schema project's mission and values, be respectful of our code of conduct, and continue to be active in the community.
+
+## Recognising our Ambassadors from the community
+
+We easily identified several community members to invite into the Ambassadors Program. The TSC and full time staff spend a LOT of time in our community spaces, so even if we don't always reply or comment, we do notice those who are active and helpful.
+
+When I first started the JSON Schema Slack server, the numbers were very few. Gradually, it grew. The core team answered most of the questions. I recall the feeling when I came back in the morning to find other people had answered questions from the community while I'd been asleep. I've previously [recognised](https://words.benhutton.me/2023-05-23-two-words-big-win) that saying "thank you" can be a big win, but I also know that we can do several magnitudes better.
+
+We already have several members of our community who have become ambassadors for JSON Schema, just without any sort of formal recognition. Recognising, supporting, and further enabling these individuals, and new ambassadors, is going to have a positive impact, and I can't wait to see what it unlocks for JSON Schema in the future.
+
+There's a swag kit for all accepted into the Ambassadors Program, and I can't wait to meet the individuals excited and proud to be wearing the exclusive swag at events and conferences in the future. We see you, ambassadors!
+
+## How to apply today
+
+OK, OK. If you've read all the above, maybe you haven't yet made it to the [Ambassador Program documentation file in our community repo](https://github.com/json-schema-org/community/tree/main/programs/ambassadors). That would be your next stop if you're thinking about applying to become an ambassador.
```

Comment:
Personal taste: I prefer shorter links because I find the link formatting difficult to read or distracting.  I'd move "in our community repo" out of the link.  (Keep the text, just not linked.)

Reply:
I created this url using a redirect: https://json-schema.org/ambassadors

Reply:
I'm not talking about the URL.  I'm talking about the link text.

Reply:
Still the new url is better.